### PR TITLE
Update README.md with more support options

### PR DESCRIPTION
**Pull request by suricactus**
As they highlighted here, we never promoted our ideas platform. https://github.com/opengisch/QField/discussions/5393#discussioncomment-9878195

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +18,10 @@ QField and QFieldCloud documentation is deployed [here](https://docs.qfield.org)
 
 ## Feature requests and issue reports
 
-If you are interested what are the new upcoming developments, or you want to request a feature, please visit our ideas platform at [ideas.qfield.org](https://ideas.qfield.org).
-There you can either submit a new feature request or upvote an already existing one.
-Alternatively, you can speed up developments by sending us an email at sales@qfield.cloud.
+If you are interested in upcoming developments, or you want to suggest a new a feature, please visit our ideas platform at [ideas.qfield.org](https://ideas.qfield.org). Here, you can submit a new request or upvote existing ones.
```
Comment (m-kuhn) [inline README.md:21]: A new a feature ...

File: README.md [hunk 2]
```diff
@@ -15,8 +16,19 @@ If you're interested in quickly getting up and running, we suggest subscribing t
 QField and QFieldCloud documentation is deployed [here](https://docs.qfield.org).
 
 
+## Feature requests and issue reports
+
+If you are interested in upcoming developments, or you want to suggest a new a feature, please visit our ideas platform at [ideas.qfield.org](https://ideas.qfield.org). Here, you can submit a new request or upvote existing ones.
```
Comment (suricactus) [inline README.md:21]:
```suggestion
If you are interested in upcoming developments, or you want to suggest a new feature, please visit our ideas platform at [ideas.qfield.org](https://ideas.qfield.org). Here, you can submit a new request or upvote existing ones.
```

File: README.md (hunk 1 above)
Comment (suricactus) [reply README.md]: Fixed.

**Comment by mbernasocchi** [comment]
@suricactus LGTM

**Review by mbernasocchi — APPROVED** [review]

**Pull request merged**